ddtrace/tracer/spancontext.go

```diff
@@ -23,6 +23,8 @@ var _ ddtrace.SpanContext = (*spanContext)(nil)
 // spawn a direct descendant of the span that it belongs to. It can be used
 // to create distributed tracing by propagating it using the provided interfaces.
 type spanContext struct {
+	updated bool // updated is tracking changes for priority / origin / x-datadog-tags
```

Comment:
This needs to be set somewhere in the code, right? i.e. in the places where it changes?

Reply:
Will set this to true in Extract method, which is absent in this PR